docs/addons.md

```diff
@@ -42,11 +42,11 @@ kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addon
 Please note that kops installs a Route53 DNS controller automatically (it is required for cluster discovery).
 The functionality of the route53-mapper overlaps with the dns-controller, but some users will prefer to
 use one or the other.
-[README for the included dns-controller](https://github.com/kubernetes/kops/blob/master/dns-controller/README.md)
+[Official README for Addon](https://github.com/wearemolecule/route53-kubernetes/blob/master/README.md)
```

Comment:
We may want to include the README link for the Route53-mapper, but we shouldn't remove the link to the DNS controller README.